engine/lib/import_thir.ml

```diff
@@ -257,6 +257,9 @@ end) : EXPR = struct
       | Ge -> Core__cmp__PartialOrd__ge
       | Gt -> Core__cmp__PartialOrd__gt
       | Eq -> Core__cmp__PartialEq__eq
+      | AddWithOverflow | SubWithOverflow | MulWithOverflow ->
+          assertion_failure (Span.to_thir span)
+            "Overflowing binary operators are not suppored"
```

Comment:
The new `WithOverflow` operators are variants that return `(T, bool)` where the bool indicates whether overflow happened. This replaces the old `CheckedBinaryOp` variant. Pretty sure this only shows up in MIR so you should be fine. That might be true of `Cmp` too actually.